velox/functions/prestosql/aggregates/ArbitraryAggregate.cpp

```diff
@@ -213,6 +245,35 @@ class NonNumericArbitrary : public exec::Aggregate {
     });
   }
 
+  bool supportAddRawClusteredInput() const override {
+    return true;
+  }
+
+  void addRawClusteredInput(
+      char** groups,
+      const folly::Range<const vector_size_t*>& groupBoundaries,
+      const std::vector<VectorPtr>& args) override {
+    VELOX_CHECK(clusteredInput_);
+    decoded_.decode(*args[0]);
+    VELOX_DCHECK(copyRanges_.empty());
+    VELOX_DCHECK(accumulators_.empty());
+    vector_size_t groupStart = 0;
+    for (auto groupEnd : groupBoundaries) {
+      auto* accumulator =
+          value<ClusteredNonNumericAccumulator>(groups[groupEnd - 1]);
+      if (!accumulator->vector) {
```

Comment:
Can you help me understand this check? Why 'else' branch is missing? When would this check return 'false'?

Reply:
Wondering why would this check return false? Would this happen than a group continues into the next batch?

Reply:
Yes if a group is crossing multiple batches

Reply:
Got it. Would it be possible to add a short comment to clarify?